prisma/seed.ts

```diff
@@ -0,0 +1,285 @@
+      // "Chai",
+      // "Kubernetes",
+      // "Terraform",
+      // "GraphQL",
+      // "MongoDB",
+    ];
+    const platformArray = [
+      "twitter",
+      "linkedIn",
+      "instagram",
+      "github",
+      "dribble",
+      "facebook",
+    ];
+
+    // USERS
+    for (let i = 0; i < 5; i++) {
+      const randomGoalsCount = Math.floor(Math.random() * 6) + 1;
+      const goals = Array.from({ length: randomGoalsCount }, () => ({
+        name: faker.lorem.sentence(),
+        isComplete: i % 2 === 0,
+      }));
+      const socialMedia = platformArray.map((platform) => ({
+        username: faker.internet.userName(),
+        type: platform,
+        link: faker.internet.url(),
+      }));
+      const password = "password123";
+      const hashedPassword = bcryptjs.hashSync(password, 10);
+
+      const user = await prisma.user.create({
+        data: {
+          name: `${faker.person.firstName()} ${faker.person.lastName()}`,
+          email: faker.internet.exampleEmail(),
+          password: hashedPassword,
+          image: faker.image.urlLoremFlickr({ category: "avatar" }),
+          location: faker.location.city(),
+          portfolio: faker.internet.url(),
+          role: faker.helpers.enumValue(Role),
+          goals: {
+            create: goals,
+          },
+          knowledgeLevel: faker.helpers.arrayElements(knowledgeLevels),
+          techStack: faker.helpers.arrayElements(tech),
+          socialMedia: {
+            create: socialMedia,
+          },
+          availability: i % 2 === 0,
+        },
+      });
+      emails.push(user.email);
+    }
+
+    // DEMO USER
+    const password = "demouser!2#";
+    const hashedPassword = bcryptjs.hashSync(password, 10);
+    const demoUserEmail = "demouser@email.com";
+
+    const demoUserSocialMedia = [
+      {
+        username: "duser",
+        type: "twitter",
+        link: faker.internet.url(),
+      },
+      {
+        username: "demo.user",
+        type: "instagram",
+        link: faker.internet.url(),
+      },
+      {
+        username: "d.dribbles",
+        type: "dribble",
+        link: faker.internet.url(),
+      },
+    ];
+
+    const demoUser = await prisma.user.create({
+      data: {
+        name: "Demo User",
+        email: demoUserEmail,
+        password: hashedPassword,
+        image: faker.image.urlLoremFlickr({ category: "avatar" }),
+        location: "San Francisco",
+        onboardingStatus: 5,
+        portfolio: "https://demouserportfolio.com",
+        role: Role.USER,
+        goals: {
+          create: [
+            { name: "Build a personal portfolio website", isComplete: false },
+            { name: "Contribute to open-source projects", isComplete: false },
+            { name: "Learn more about cloud computing", isComplete: true },
+            { name: "Understand the principles of DevOps", isComplete: false },
+          ],
+        },
+        //TODO: techStack: ["reactjs", "Python", "Django", "GraphQL"],
```

Comment:
This needs a bigger update in the frontend - associating each tech to its icon with the correct styles